components/PreferenceList/index.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import PreferenceCard from "../PreferenceCard";
+
+export default function PreferenceList({ preferences }) {
+  return (
+    <>
+      {preferences.length === 0 ? (
+        <StyledArticle>
+          <p>
+            Start adding your <StyledSpan>first</StyledSpan> preference!
+          </p>
+        </StyledArticle>
```

Comment:
I think you could also style the `p` element directly. No extra element necessary for this. :)